TaskSharper.DataAccessLayer.Google/Authentication/GoogleAuthentication.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Calendar.v3;
+using Google.Apis.Util.Store;
+
+namespace TaskSharper.DataAccessLayer.Google.Authentication
+{
+    public static class GoogleAuthentication
```

Comment:
Consider making this class and methods not static for better testability

Reply:
Done